src/HashBus.Twitter.Monitor.Simulator/App.cs

```diff
@@ -1,23 +1,39 @@
 namespace HashBus.Twitter.Monitor.Simulator
 {
+    using System.Threading;
+    using System.Threading.Tasks;
+    using HashBus.Application.Commands;
     using HashBus.NServiceBusConfiguration;
     using NServiceBus;
     using NServiceBus.Persistence;
 
     class App
     {
-        public static void Run(string nserviceBusConnectionString, string hashtag, string endpointName)
+        public static async Task Run(string nserviceBusConnectionString, string hashtag, string endpointName, string applicationAddress)
         {
-            var busConfiguration = new BusConfiguration();
-            busConfiguration.EndpointName(endpointName);
-            busConfiguration.UseSerialization<JsonSerializer>();
-            busConfiguration.EnableInstallers();
-            busConfiguration.UsePersistence<NHibernatePersistence>().ConnectionString(nserviceBusConnectionString);
-            busConfiguration.ApplyMessageConventions();
+            var endpointConfiguration = new EndpointConfiguration(endpointName);
+            endpointConfiguration.UseSerialization<JsonSerializer>();
+            endpointConfiguration.EnableInstallers();
+            endpointConfiguration.UsePersistence<NHibernatePersistence>().ConnectionString(nserviceBusConnectionString);
+            endpointConfiguration.ApplyMessageConventions();
+            endpointConfiguration.ApplyErrorAndAuditQueueSettings();
 
-            using (var bus = Bus.Create(busConfiguration).Start())
+            var transportExtensions = endpointConfiguration.UseTransport<MsmqTransport>();
+            var routing = transportExtensions.Routing();

```

Comment:
Please insert a blank line before this line (for consistency with other endpoints).